impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -210,6 +211,23 @@ else if (SYSPROPS_TOKEN.equalsIgnoreCase(location)) {
             clientConfig.setProxyPassword(props.get(DEFAULT_CLIENT_PROXY_PASSWORD_PROPERTY_NAME));
         }
 
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_AUTHORIZATION_MODE_PROPERTY_NAME))) {
+            clientConfig.setAuthorizationMode(props.get(DEFAULT_CLIENT_AUTHORIZATION_MODE_PROPERTY_NAME));
+        }
+
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_CLIENT_ID_PROPERTY_NAME))) {
+            clientConfig.setClientId(props.get(DEFAULT_CLIENT_CLIENT_ID_PROPERTY_NAME));
+        }
+
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_SCOPES_PROPERTY_NAME))) {
```

Comment:
@bretterer what is the format of the SCOPES in the yaml? a string or a list of strings?

Reply:
@bdemers its a list of strings, I believe.

Reply:
It's a list of strings.

Reply:
@laura-rodriguez how should it be formatted for the equivalent environment variable?
comma-separated (and individually trimmed?)
meaning both:
```
one,two,three,four
```
and
```
one, two, three,four
```
would work?
Or space separated to _match_ the OAuth spec?

Reply:
I have this one pending in the .NET SDK. I believe that @bretterer is supporting scopes via env var, but I don't remember the expected format.

Reply:
@bdemers I raised this question in today's scrum. Env variable string is space separated (which goes with the way scopes are accepted by OAuth API).